nautobot_ssot/integrations/vsphere/models.py

```diff
@@ -96,6 +96,10 @@ class SSOTvSphereConfig(PrimaryModel):  # pylint: disable=too-many-ancestors
         max_length=CHARFIELD_MAX_LENGTH, verbose_name="Default Cluster Type", default="VMWare vSphere"
     )
     is_saved_view_model = False
+    is_dynamic_group_associable_model = False
```

Comment:
These are PrimaryModel's for Config, that probably should have been BaseModel. I don't see a reason to have saved_views, dynamic_groups, contacts or data compliance. In addition, the Data Compliance tab would error when accessing.